### PR TITLE
Add support for blade component registration in packages

### PR DESCRIPTION
**Pull request by pdbreen**

### CODE REVIEW

File: src/SupportServiceProvider.php [hunk 1]
```diff
@@ -11,6 +13,15 @@ public function configureTipoffPackage(TipoffPackage $package): void
             ->hasConfigFile('tipoff');
     }
 
+    public function bootingPackage()
+    {
+        parent::bootingPackage();
+
+        Blade::directive('tipoffMoney', function ($amount) {
```
Comment (pdbreen) [inline src/SupportServiceProvider.php]: @drewroberts - note the `tipoff` prefix on the directive registration.  Seemed a bit aggressive to register a common term without some prefix.

**Comment by pdbreen** [comment]
Flagged as don't merge.  Want to fix tmp dir issue when running tests on Windows before letting this in.

**Comment by pdbreen** [comment]
Ok - this is good to go now.  Unfortunately, the fix for the temp file issue in Windows Github action testing is both in this package and in test-support since this package can't pull in test support.

**Review by drewroberts — APPROVED** [review]
Thank you! 💰

**Pull request merged**